### PR TITLE
fix problems found in real world app

### PR DESCRIPTION
**Pull request by BryanCrotaz**
Failed when running on my code base, this commit fixes the AST so that it runs on my codebase

### CODE REVIEW

File: addon/modifiers/create-ref.js [hunk 1]
```diff
@@ -12,9 +12,11 @@ export default class RefModifier extends Modifier {
...
     setGlobalRef(getOwner(this));
   }
-  mutationObserverOptions = {
-    attributes: true,
-    characterData: true,
+  // to minimise overhead, user should be specific about
+  // what they want to observe
+  defaultMutationObserverOptions = {
+    attributes: false,
+    characterData: false,
```
Comment (lifeart) [inline addon/modifiers/create-ref.js:19]: looks like it's overlapping with https://github.com/lifeart/ember-ref-bucket/pull/14/files

File: lib/ref-transform.js [hunk 2]
```diff
@@ -36,7 +36,9 @@ module.exports = class SetTransform {
            {{${originalName}}} should be {{${typos[originalName]}}}
         `);
       }
-      if (!originalName.includes('ref')) {
+      if (!originalName ||
```
Comment (lifeart) [inline lib/ref-transform.js:39]: could you mention template itself for this case?

Comment (BryanCrotaz) [reply lib/ref-transform.js:39]: Sorry don't understand what you mean

File: addon/modifiers/create-ref.js (hunk 1 above)
Comment (BryanCrotaz) [reply addon/modifiers/create-ref.js:19]: Let me know when you've merged it and I'll merge up

File: lib/ref-transform.js (hunk 2 above)
Comment (lifeart) [reply lib/ref-transform.js:39]: sorry, I mean, could you add template example with `originalName` = false | undefined or such?

**Comment by lifeart** [comment]
@BryanCrotaz aaaa, sorry, wrong closing, https://github.com/lifeart/ember-ref-bucket/pull/14/files merged, could you update current MR to follow-up?

Can't reopen it manually because of 
image

File: lib/ref-transform.js (hunk 2 above)
Comment (BryanCrotaz) [reply lib/ref-transform.js:39]: no clue how to reproduce, this was in a real world app with hundreds of templates, and the error didn't allow me to see which template was causing it.

**Comment by BryanCrotaz** [comment]
replaced by #16

**Pull request closed without merging**